### ISSUE TITLE
Error within my game but not on seperate project?

### ISSUE THREAD

**Issue by Elit3d** [InvexGames/MaterialUI#42]
https://gyazo.com/91f6e926cc53cdb9397e155d8573694a not too sure what can be causing this.

**Comment by nah0y** [comment]
Hello.
What is the Unity version you're using?
Minimum must be 4.6.2

**Comment by Elit3d** [comment]
I am using the latest 4.6, as I said it works on a seperate project but for some reason not in my game project

**Comment by nah0y** [comment]
Maybe you have a class named Toggle in your project

**Comment by Elit3d** [comment]
Hm maybe. I'll have a look around. I am also using a custom inputmanager that changes the standalone input to another script maybe that coulthee it

**Comment by InvexGames** [comment]
Any luck?

**Comment by Elit3d** [comment]
No luck, im assuming its InputManager plugin as it requires a standalone input script change.

**Comment by nah0y** [comment]
Try removing InputManager to see if it's the cause maybe? Just to make sure.
And sure that you don't have another class called Toggle?

**Comment by Elit3d** [comment]
Yes I had a class called Toggle got rid of it and it worked. Now a question for the dropdown scrolling or just the UI stuff in general, how well can it handle controller support as well as keyboard and mouse support?

**Comment by Elit3d** [comment]
Also because I am using another inputmanager where would I go to make the hover events work for things like the dropdown etc as I will need to change some code to register my mouse

**Comment by InvexGames** [comment]
Hi! Thanks for bringing this issue to my attention :)

I'm now working on MaterialUI in a private repo, adding features and the likes for an official release on the Asset Store - this means that I won't be adding/fixing anything more in this repo (sorry). Feel free to fork this one if you'd like to tackle to issue yourself, and/or send me an email at invex@invexgames.com if you want me to let you know when it's available.

Cheers,
Declan.
